src/styles/blocks/stars.css

```
@@ -0,0 +1,48 @@
+.stars {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+}
+
+.stars__star {
+  background-image: url(../../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars.stars--1 .stars__star:first-child {
+  background-image: url('../../images/star-active.svg');
+}
+
+.stars.stars--2 .stars__star:first-child,
+.stars.stars--2 .stars__star:nth-child(2) {
+  background-image: url('../../images/star-active.svg');
+}
+
+.stars.stars--3 .stars__star:first-child,
+.stars.stars--3 .stars__star:nth-child(2),
+.stars.stars--3 .stars__star:nth-child(3) {
+  background-image: url('../../images/star-active.svg');
+}
+
+.stars.stars--4 .stars__star:first-child,
+.stars.stars--4 .stars__star:nth-child(2),
+.stars.stars--4 .stars__star:nth-child(3),
+.stars.stars--4 .stars__star:nth-child(4) {
+  background-image: url('../../images/star-active.svg');
+}
+
+.stars.stars--5 .stars__star:first-child,
+.stars.stars--5 .stars__star:nth-child(2),
+.stars.stars--5 .stars__star:nth-child(3),
+.stars.stars--5 .stars__star:nth-child(4),
+.stars.stars--5 .stars__star:nth-child(5) {
```

```suggestion
.stars--1 .stars__star:nth-child(-n + 1),
.stars--2 .stars__star:nth-child(-n + 2),
.stars--3 .stars__star:nth-child(-n + 3),
.stars--4 .stars__star:nth-child(-n + 4),
.stars--5 .stars__star:nth-child(-n + 5) {
```
You can simplify and combine selectors. https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child